### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by Namrata-J**
#2 Have added components in my dummy folder structure of productlisting page, added header, sidebar, product section(wherein all the products are being listed)

have seperated out components, so folders to check are:
- FilterableForm.js (wherein all the filters are seperated out as a single component)
- SideBar.js (wherein i have imported the filterableForm component)
- FloatingBtn.js (wherein i have imported the filterable component, basically added for the responsiveness purpose)
- productDetails-context.js (wherein i have made a context for product details by fetching the data from the api of product.js)
- ProductCard.js (wherein i have seperated out the card component)

Please review it, thankyou!!

live preview: [aprize-ecom-site.netlify.app](https://aprize-ecom-site.netlify.app/)

### CODE REVIEW

File: src/components/Product-card/ProductCard.jsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                    )
+                })
+            }
+        </div>
+    );
+}
+
+export { ProductCard };
+
+{/* <div className="card-w-badge-type1 card-w-badge b-rad1">
```
Comment (ultronjs) [inline src/components/Product-card/ProductCard.jsx]: Remove the commented Code.

**Review by ultronjs — APPROVED** [review]
Else Good to Merge

File: src/components/Product-card/ProductCard.jsx (hunk 1 above)
Comment (Namrata-J) [reply src/components/Product-card/ProductCard.jsx]: thanks, for pointing that out!!

File: src/components/Product-card/productCard.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+.card-product-price {
+    width: 25%;
+}
+
+.ap_card-pp {
+    display: flex;
+    flex-direction: column;
+}
+
```
Comment (mariyasada) [inline src/components/Product-card/productCard.css:38]: can make it as utility class

File: src/contexts/productsDetail-context.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+const ProductsDetailProvider = ({ children }) => {
+    
+    const [products, setProducts] = useState([]);
+
+    useEffect(() =>
+        (async () => {
+            const ProductResponse = await axios.get("api/products")
+            setProducts(ProductResponse.data.products)
+        })(), []
+    );
```
Comment (mariyasada) [inline src/contexts/productsDetail-context.js:15]: you can destructure productResponse here like {productResponse}

**Review by mariyasada — APPROVED** [review]
good to go

File: src/contexts/productsDetail-context.js (hunk 3 above)
Comment (Namrata-J) [reply src/contexts/productsDetail-context.js:15]: okay, will try doing it the suggested way, thanks for the suggestion!!

**Pull request merged**